src/app/community/_components/UserProfileCard.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+
+import classNames from 'classnames/bind';
+import { forwardRef, useEffect, useState } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import Image from 'next/image';
+
+import { getOthersInfo } from '@/api/usersAPI';
+import { SpinLoading, keydeukProfileImg } from '@/public/index';
+
+import styles from './UserProfileCard.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface UserProfileCardProps {
+  isOpenProfileCard: boolean;
+  userId: number;
+  positionTop?: number;
+}
+export default forwardRef<HTMLDivElement, UserProfileCardProps>(function UserProfileCard(
+  { isOpenProfileCard, userId, positionTop },
+  ref,
+) {
+  const {
+    data: userInfo,
+    refetch,
+    isFetching,
+  } = useQuery({
+    queryKey: ['clickedUserInfo'],
+    queryFn: () => getOthersInfo(userId),
+    enabled: false,
+  });
+  const [isAboveProfile, setIsAboveProfile] = useState(false);
+
+  useEffect(() => {
+    const VIEWPORT_HEIGHT = window.innerHeight;
+
+    if (isOpenProfileCard) {
+      refetch();
+      if (positionTop && positionTop > VIEWPORT_HEIGHT / 2) {
+        setIsAboveProfile(true);
+      }
+    }
+  }, [isOpenProfileCard, userId, refetch, positionTop, isAboveProfile]);
+
+  return positionTop && positionTop === 0 ? null : (
```

Comment:
```suggestion
  return positionTop && positionTop === 0 ? null : (
```
여기서 positionTop이 없으면 그냥 null처리 하셔서 아무것도 안보여주시는식으로 설계하셨는데
아무것도 없는 데이터를 보여줘도 괜찮은 컴포넌트인가요?

Reply:
그게 아니라면 사용처에서 useState로 관리하는게 더 낫지않을까요?

Reply:
data 가 null 값이어도 컴포넌트 내부 내용의 구조?는 그대로 유지 되기 때문에 아무것도 안보여줘도 괜찮을 것 같다고 생각했습니다! 아니면 데이터가 null 값이었을 때의 추가적인 처리를 선호하시는건가요?